build/3-apillon-api.md

```diff
@@ -1158,3 +1158,714 @@ curl --location --request GET "https://api.apillon.io/hosting/websites/:websiteU
+| limit   | Number of transactions on a page (default: 20).                                                         | false    |
+| orderBy | One or multiple properties, separated by a comma, used to order data.                                   | false    |
+| desc    | `Boolean` values, mapped to the index of the `orderBy` parameter. Defaults to false.                    | false    |
+
+#### Response Fields
+
+| Name              | Type        | Description                                                 |
+|-------------------|-------------|-------------------------------------------------------------|
+| status            | `number`    | Apillon internal/database transaction status.               |
+| chainId           | `number`    | Blockchain id on which you want to release your collection. |
+| transactionType   | `number`    | Transaction type.                                           |
+| transactionStatus | `number`    | Transaction status                                          |
+| transactionHash   | `number`    | Transaction hash/id.                                        |
+| updateTime        | `DateTime`  | Transaction last update time.                               |
+
+  </div>
+  <div class="split_side">
+
+<CodeGroup>
+    <CodeGroupItem title="cURL" active>
+
+```sh
+curl --location 'https://api.apillon.io/nfts/collections/:uuid/transactions' \
+--header 'Authorization: Basic :credentials'
+```
+
+  </CodeGroupItem>
+</CodeGroup>
+<CodeGroup>
+  <CodeGroupItem title="Response">
+
+```json
+{
+    "id": "b5935c73-204d-4365-9f9a-6a1792adab5b",
+    "status": 200,
+    "data": {
+        "items": [
+            {
+                "id": 7,
+                "status": 5,
+                "chainId": 1287,
+                "transactionType": 1,
+                "transactionStatus": 1,
+                "transactionHash": "0xb988c8cda7ec8b441611b208360e0aace9c294e1ca5672375b21e815890a54d1",
+                "updateTime": "2023-06-13T10:15:58.000Z"
+            }
+        ],
+        "total": 1
+    }
+}
+```
+
+</CodeGroupItem>
+</CodeGroup>
+  </div>
+</div>
+
+##### Statuses
+
+| Number | Description         |
+|--------|---------------------|
+| 1      | Draft               |
+| 2      | Incomplete          |
+| 5      | Active              |
+| 8      | Marked for deletion |
+| 9      | Deleted             |
+
+##### Transaction Types
+
+| Number | Description                 |
+|--------|-----------------------------|
+| 1      | Deploy Contract             |
+| 2      | Transfer Contract Ownership |
+| 3      | Mint NFT                    |
+| 4      | Set Collection Base URI     |
+| 5      | Burn NFT                    |
+
+##### Transaction Status
+
+| Number | Description |
+|--------|-------------|
+| 1      | Pending     |
+| 2      | Confirmed   |
+| 3      | Failed      |
+| 4      | Error       |
+
+
+
+### Create NFT Collection
+
+> API that creates NFT collection and deploys it on selected network.
```

Comment:
Would expand this description to explain functunalities of:
- drop
- revokable
- soulbound
- royalties